### PR TITLE
Users API works with User domain model

### PR DESCRIPTION
**Pull request by biern**

### CODE REVIEW

File: horse/models/user.py [hunk 1]
```diff
@@ -1,5 +1,6 @@
 class User:
-    def __init__(self, name, followed_users=None, liked_movies=None):
+    def __init__(self, id, name, followed_users=None, liked_movies=None):
+        self.id = id
```
Comment (pgrzesik) [inline horse/models/user.py]: id is a builtin, maybe we should change it ?

File: horse/users.py [hunk 2]
```diff
@@ -10,14 +12,21 @@
 
 
 def get_user_by_id(user_id):
-    return [u for u in users if u['id'] == user_id][0]
+    return [u for u in users if u.id == user_id][0]
+
+
+def jsonify_user(user):
```
Comment (pgrzesik) [inline horse/users.py:18]: maybe this should be a method of User model ?

**Comment by pgrzesik** [comment]
review done

File: horse/models/user.py (hunk 1 above)
Comment (biern) [reply horse/models/user.py]: Agree. What about `_id`?

File: horse/users.py (hunk 2 above)
Comment (biern) [reply horse/users.py:18]: This method is API specific and I'd keep it away from domain models. I hope that soon we'll manage to implement robust serialization, eg. `marshmallow`

File: horse/models/user.py (hunk 1 above)
Comment (biern) [reply horse/models/user.py]: ok, we're settled on `pk`

File: horse/users.py (hunk 2 above)
Comment (pgrzesik) [reply horse/users.py:18]: fair enough, using marshmallow seems like a very good idea

**Pull request merged**